enforcer/datapath.go

```diff
@@ -330,10 +371,12 @@ func (d *datapathEnforcer) StartNetworkInterceptor() {
 		}
 
 		go func(i uint16) {
```

Comment:
just for clarity, it's best to not use the same variable names as the enclosing function. This could lead to weird error in case of mistakes. You could call it `index`